packages/babel-preset/index.js

```diff
@@ -0,0 +1,37 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2023 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+/* eslint-env node */
+
+module.exports = () => {
+  return {
+    // TODO: Add @realm/babel-plugin here
+    overrides: [
+      {
+        plugins: [
+          [
+            require("@babel/plugin-transform-flow-strip-types"),
```

Comment:
Shouldn't we use the TS plugin rather than flow? https://babeljs.io/docs/babel-plugin-transform-typescript#allowdeclarefields

Reply:
I thought so too, but that's not the one that complained though ... 🤔

Reply:
This override is patching the plugin which is pulled in via https://github.com/realm/realm-js/blob/bindgen/integration-tests/environments/react-native/babel.config.js#L19 (which I expect any React Native app to use) .. but we should probably assert that it's already loaded as a plugin. To be honest I find the Babel config override and preset APIs are super confusing and would love some help on that at some point.